definitions/variable/grids infrustructure/electricity_grid.yaml

```
@@ -0,0 +1,111 @@
+- Network|Electricity|Capacity:
+    description: Capacity of the line
+    unit: MW
+- Network|Electricity|Resistance:
+    description: Resistance of a line
+    unit: Ohm
+- Network|Electricity|Reactance:
+    description: Reactance of a line
+    unit: Ohm
+- Network|Electricity|Capacitance:
+    description: Parallel capacitance of a line
+    unit: nF
+- Capacity|Electricity|Fossil|Gas Turbine:
+    description: Installed (available) capacity to generate electricity from fossil fuel gas turbines
+    unit: GW
+- Capacity|Electricity|Fossil|Combined Cycle:
+    description: Installed (available) capacity to generate electricity from fossil fuel combined cycle plants
+    unit: GW
+- Capacity|Electricity|Gas|Combined Cycle:
+    description: Installed (available) capacity to generate electricity from natural gas combined cycle plants
+    unit: GW
+- Capacity|Electricity|Gas|Gas Turbine:
+    description: Installed (available) capacity to generate electricity from natural gas gas turbines
+    unit: GW
+- Capacity|Electricity|Gas|Integrated Gasification Combined Cycle:
+    description: Installed (available) capacity to generate electricity from integrated gasification combined cycle plants
+    unit: GW
+- Capacity|Electricity|Gas|Steam Turbine:
+    description: Installed (available) capacity to generate electricity from natural gas steam turbines
+    unit: GW
```

You could simplify this file by using a tag-list, so creating a separate file `tag_power_turbine_types.yaml` (or similar) that has
```yaml
- Turbine Type:
    - Combined Cycle:
        description: combined-cycle (CC)
    - Gas Turbine:
         description: gas turbines (GT)
    - Integrated Gasification Combined-Cycle:
        description: integrated gasification combined cycle plants (IGCC)
    - Steam Turbine:
        description: steam turbines
```
And then simplify this section
```suggestion
- Capacity|Electricity|Gas|{Turbine Type}:
    description: Installed (available) capacity to generate electricity with natural gas
      using {Turbine Type} plants
    unit: GW
```
Same for all other variables that use all combinations of turbines.

This produces a much more concise and readable variable list file. The nomenclature package will automatically create all permutations during scenario validation.

---

It's done now.